src/pages/AppList.svelte

```diff
@@ -18,6 +18,7 @@
 
   onMount(async () => {
     apps = await fetchJSON(URL);
+    console.log("App name", apps);
```

Comment:
I think you forgot to take this out?

Reply:
oh yeah! worked on it now. Thank you @Iinh